src/components/views/beacon/displayStatus.ts

```diff
@@ -25,14 +25,17 @@ export enum BeaconDisplayStatus {
 export const getBeaconDisplayStatus = (
     isLive: boolean,
     latestLocationState?: BeaconLocationState,
-    error?: Error): BeaconDisplayStatus => {
+    error?: Error,
+    waitingToStart?: boolean): BeaconDisplayStatus => {
```

Comment:
it's not clear where the function definition starts and code begins. Under the inbound code style:
```suggestion
    waitingToStart?: boolean,
): BeaconDisplayStatus => {
```